app/src/main/java/com/github/se/gomeet/ui/mainscreens/Trends.kt

```diff
@@ -106,43 +128,37 @@ fun Trends(
         Column(
             verticalArrangement = Arrangement.Top,
             horizontalAlignment = Alignment.CenterHorizontally,
-            modifier = Modifier.padding(innerPadding)) {
+            modifier = Modifier.padding(innerPadding).verticalScroll(rememberScrollState())) {
               Spacer(modifier = Modifier.height(5.dp))
-              GoMeetSearchBar(nav, query, NavBarUnselected, Color.DarkGray)
+              GoMeetSearchBar(
+                  nav,
+                  query,
+                  MaterialTheme.colorScheme.primaryContainer,
+                  MaterialTheme.colorScheme.tertiary)
               Spacer(modifier = Modifier.height(5.dp))
 
+              SortButton(eventList)
+
               if (!eventsLoaded.value) {
                 LoadingText()
               } else {
+                Spacer(modifier = Modifier.height(5.dp))
+                // TODO: Use the top 5 events instead
+                EventCarousel(eventList.take(5), nav)
 
-                Column(modifier = Modifier.verticalScroll(rememberScrollState()).fillMaxSize()) {
-                  Text(
-                      text = "All Events",
-                      style =
-                          TextStyle(
-                              fontSize = 20.sp,
-                              lineHeight = 16.sp,
-                              fontFamily = FontFamily(Font(R.font.roboto)),
-                              fontWeight = FontWeight(1000),
-                              color = DarkCyan,
-                              textAlign = TextAlign.Center,
-                              letterSpacing = 0.5.sp,
-                          ),
-                      modifier = Modifier.padding(10.dp).align(Alignment.Start))
-
+                Column(modifier = Modifier.fillMaxSize()) {
+                  // TODO: Remove the top 5 events from the list
```

Comment:
I don't understand why the top 5 events need to be removed from the list

Reply:
Maybe not to have them listed twice (both in the carousel and list) but that's to be determined later

Reply:
Oh ok I see, well if it's just 5 events that are listed twice I think it should be fine